boards/xtensa/intel_s1000_crb/support/create_board_img.py

```diff
@@ -0,0 +1,221 @@
+#!/usr/bin/env python3
+#
+# Copyright (c) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+# This script will add specific headers and FW load message to tell
+# intel_s1000 ROM bootloader to load the firmware
+# Usage python3 ${ZEPHYR_BASE}/scripts/create_board_img.py
+#                    -i in_file.bin -o out_file.bin
+#                    [-m|--no_sram] [-c|--no_l1cache] [-t|--no_tlb]
+#                    [-x|--no_exec] [-s|--no_sha] [-k|--clk_sel]
+
+import argparse
+import os
+import sys
+import struct
+
+help_text="""
+The flash memory needs to have specific headers and fw load message to tell
+the ROM bootloader to load firmware.
+
+To run script use command:
+
+create_spi_flash.py -i in_file.bin -o out_file.bin
```

Comment:
The file name of the script is `create_board_img.py`

Reply:
Thanks for catching.